src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const valueOne = Big(numberOne);
+  const valueTwo = Big(numberTwo);
+  let total = 0;
+
+  switch (operation) {
+    case '+':
+      total = valueOne.plus(valueTwo);
+      break;
+    case '-':
+      total = valueOne.minus(valueTwo);
+      break;
+    case 'x':
+      total = valueOne.times(valueTwo);
+      break;
+    case '÷':
+      if (valueTwo === 0) {
+        total = "Can't divide by zero";
+      } else {
+        total = valueOne.div(valueTwo);
+      }
+      break;
+    case '%':
+      total = valueOne.div(100);
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](http://mikemcl.github.io/big.js/#mod)

Example: case `%` will be -> `somevalue.mod(somevalue)`

Reply:
Thanks for the tip, but I think, in this case, we are not looking for the remainder, or modulo, instead this symbol in the calculator means the percentage of a number.

Reply:
@ferbaco86 Yes, we are not looking remainder, in this case, you only supply the first digit right?
`example: click `10` then `%` = 0.01
`

Reply:
Yes, only the first one